jgiven-spock2/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java-library'
+}
+
+apply plugin: 'groovy'
+
+description = "Module for writing JGiven tests with Spock 2"
+
+ext {
+    spock2_version = '2.0-groovy-3.0'
+    groovy_version = '3.0.9'
+}
+
+dependencies {
+    api project(':jgiven-junit')
+
+    implementation "org.codehaus.groovy:groovy:$groovy_version"
+    implementation "org.spockframework:spock-core:$spock2_version"
+    implementation "org.spockframework:spock-junit4:$spock2_version"
```

Comment:
Question: wouldn't it make sense to try to align the spock2 package with JUnit 5 directly? If I recall correctly Spock2 is JUnit5 based and upgrading the published jgiven package later should be significantly harder.

Reply:
Spock2 uses JUnit Platform used in JUnit5, but it does not use JUnit5. Therefore can not use the Junit5 Extensions.

At the end, it is an implementation detail for the user of the library. The user will not see that internally we are using JUnit4 Rules, but we can migrate to native Spock at some point in the future.

Using Spock2 allows users of the library to migrate to JDK17